tests/integration/suite/scheduler/helm/ha.go

```diff
@@ -15,114 +15,142 @@ package helm
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-ports", "dapr-scheduler-server-0=2379,dapr-scheduler-server-1=2379,dapr-scheduler-server-2=2379")
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-http-ports", "dapr-scheduler-server-0=2330,dapr-scheduler-server-1=2330,dapr-scheduler-server-2=2330")
+	})
+
+	// disabled scheduler (replica count 0)
+	t.Run("get_default_replicas_disabled", func(t *testing.T) {
+		var sts appsv1.StatefulSet
+		bs, err = io.ReadAll(b.helmDisableStdout)
+		require.NoError(t, err)
+		require.NoError(t, yaml.Unmarshal(bs, &sts))
+		require.NotNil(t, *sts.Spec.Replicas)
+		require.Equal(t, int32(0), *sts.Spec.Replicas)
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--replica-count", "0")
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--initial-cluster", "")
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-ports", "")
+		requireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-http-ports", "")
 	})
 
 	// namespaced
 	t.Run("initial_cluster_has_all_instances_dapr_system_namespace", func(t *testing.T) {
 		var stsNamespaced appsv1.StatefulSet
-		require.NoError(t, yaml.Unmarshal(b.helmNamespaced.GetStdout(), &stsNamespaced))
-		helm.RequireArgsValue(t, stsNamespaced.Spec.Template.Spec.Containers[0].Args, "--initial-cluster",
+		bs, err = io.ReadAll(b.helmNamespacedStdout)
+		require.NoError(t, err)
+		require.NoError(t, yaml.Unmarshal(bs, &stsNamespaced))
+		requireArgsValue(t, stsNamespaced.Spec.Template.Spec.Containers[0].Args, "--initial-cluster",
 			"dapr-scheduler-server-0=http://dapr-scheduler-server-0.dapr-scheduler-server.dapr-system.svc.cluster.local:2380,"+
 				"dapr-scheduler-server-1=http://dapr-scheduler-server-1.dapr-scheduler-server.dapr-system.svc.cluster.local:2380,"+
 				"dapr-scheduler-server-2=http://dapr-scheduler-server-2.dapr-scheduler-server.dapr-system.svc.cluster.local:2380")
 	})
+}
 
-	// bad path helmerr
-	t.Run("get_error_on_even_replicas", func(t *testing.T) {
-		require.Contains(t, string(b.helmErr.GetStderr()), "should be an odd number")
-	})
-
-	// disabled scheduler (replica count 0)
-	t.Run("get_default_replicas_disabled", func(t *testing.T) {
-		var sts appsv1.StatefulSet
-		require.NoError(t, yaml.Unmarshal(b.helmDisabled.GetStdout(), &sts))
-		require.NotNil(t, *sts.Spec.Replicas)
-		require.Equal(t, int32(0), *sts.Spec.Replicas)
-		helm.RequireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--replica-count", "0")
-		helm.RequireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--initial-cluster", "")
-		helm.RequireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-ports", "")
-		helm.RequireArgsValue(t, sts.Spec.Template.Spec.Containers[0].Args, "--etcd-client-http-ports", "")
-	})
+func requireArgsValue(t *testing.T, args []string, arg, value string) {
```

Comment:
shouldn't this be more global/exported somewhere as it can be used by any helm test in the future?

Reply:
I guess it could be a future refactor when needed

Reply:
Yeah, let's keep it local for now and make it exported if needed in future.